harness/determined/core/_train.py

```diff
@@ -177,20 +211,6 @@ def _get_serializable_metrics(self, metrics: Dict[str, Any]) -> Set[str]:
 
         return serializable_metrics
 
-    def report_validation_metrics(
```

Comment:
why did this have to move?

Reply:
Just to be next to the other metric functions. It was separated by a bunch of unrelated functions for some reason